NBitcoin/FeeRate.cs

```diff
@@ -56,7 +56,7 @@ public FeeRate(Money feePaid, int size)
 			if (feePaid.Satoshi < 0)
 				throw new ArgumentOutOfRangeException(nameof(feePaid), "Cannot be less than 0.");
 			if (size > 0)
-				_FeePerK = (long)((decimal)feePaid.Satoshi / (decimal)size * 1000m);
+				_FeePerK = (long)Math.Round((decimal)feePaid.Satoshi / size * 1000m, MidpointRounding.AwayFromZero);

```

Comment:
@lukasdll just curious, if the LOC said `_FeePerK = (long)((decimal)feePaid.Satoshi / (decimal)(size * 1000));` instead of `_FeePerK = (long)((decimal)feePaid.Satoshi / (decimal)size * 1000m);`, wouldn't the automatic rounding done by the cast been better? I mean, your fix is probably better than what I just said, but just thinking that the cast being done after the multiplication would have been better before

Reply:
nevermind, I was looking at the wrong cast (I guess it's the `(long)` cast that is problematic, not the `(decimal)` one)